reporter/html.js

```diff
@@ -1,4 +1,5 @@
 import QUnit from "../src/core";
+import onError from "../src/core/onerror";
```

Comment:
This file was previously not importing anything from `core` directly (other than the `QUnit` object) to ensure that the HTML Reporter stays loosely-coupled.

So, I'd like to see us just use `QUnit.onError` instead. This way, once we move to a pluggable reporter setup, the HTML Reporter is just another potential option.